### PR TITLE
Lapps formula change for homebrew

### PR DESCRIPTION
**Pull request by koncar**

### CODE REVIEW

File: Formula/lapps-cli.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ class ShipaCli < Formula
 
     platform = ""
     sha = ""
-    root_url = "https://storage.googleapis.com/shipa-client/" + version + "/shipa_"
+    root_url = "https://storage.googleapis.com/shipa-client/" + version + "/lapps_"
```
Comment (fnovikov) [inline Formula/lapps-cli.rb:9]: we need to change 
```homepage "https://shipa.io/"```
```odie "Sorry, architecture not supported: See instructions on downloading Shipa Client from https://learn.shipa.io/docs/downloading-the-shipa-client"```
```
    def install
        Dir["*"].each do |binary|
            File.rename(binary, "shipa")
        end
        bin.install "shipa"
    end
  
    test do
      system "#{bin}/shipa", "help"
    end
end```

File: Formula/lapps-cloud-cli.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@ class ShipaCloudCli < Formula
 
     platform = ""
     sha = ""
-    root_url = "https://storage.googleapis.com/shipa-client-cloud/" + version + "/shipa_"
+    root_url = "https://storage.googleapis.com/shipa-client-cloud/" + version + "/lapps_"
```
Comment (fnovikov) [inline Formula/lapps-cloud-cli.rb:9]: this file too

File: Formula/lapps-cli.rb (hunk 1 above)
Comment (fnovikov) [reply Formula/lapps-cli.rb:9]: and maybe update version and sha

Comment (koncar) [reply Formula/lapps-cli.rb:9]: fixed

File: Formula/lapps-cloud-cli.rb (hunk 2 above)
Comment (koncar) [reply Formula/lapps-cloud-cli.rb:9]: fixed

**Review by fnovikov — APPROVED** [review]

**Pull request merged**